src/main/java/savvy/core/relationship/events/DoSelectFilterRelationship.java

```diff
@@ -0,0 +1,9 @@
+package savvy.core.relationship.events;
```

Comment:
UI events should be in their corresponding UI package
see: DoSelectFilterEntity
rename the event using the DoItemAction convention